app/assets/javascripts/discourse/app/instance-initializers/message-bus.js

```diff
@@ -86,6 +95,13 @@ export default {
     // pass in a position
     const interval = setInterval(() => {
       if (document.readyState === "complete") {
+        if (
```

Comment:
There might be a more elegant way of doing this, feel free to suggest one (also in ajax.js)